### PR TITLE
fix: Add regex for web.keyman.com.localhost

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses comment on https://github.com/keymanapp/keyman/pull/10566#discussion_r1472195112

Tweaks regex so web.keyman.com.localhost and keymanweb.com.localhost correctly brings up the keymanweb site

### CODE REVIEW

File: resources/nginx.conf [hunk 1]
```diff
@@ -31,7 +31,7 @@ server {
 server {
   listen       80;
   listen  [::]:80;
-  server_name  keymanweb.com.localhost;
+  server_name  ~^(web.)?keyman(web)?.com.localhost;
```
Comment (mcdurdin) [inline resources/nginx.conf:34]: can we instead do `~^(web.keyman.com.localhost|keymanweb.com.localhost);` -- easier to read and doesn't allow web.keymanweb.com.localhost which is redundant and actually doesn't map on prod? Also do we need to escape `.`?

```suggestion
  server_name  ~^(web\.keyman\.com|keymanweb\.com)\.localhost;
```

Comment (darcywong00) [reply resources/nginx.conf]: How about `~^(web\.keyman|keymanweb).com.localhost;`

Comment (mcdurdin) [reply resources/nginx.conf]: ok

File: resources/nginx.conf [hunk 2]
```diff
@@ -31,7 +31,7 @@ server {
 server {
   listen       80;
   listen  [::]:80;
-  server_name  keymanweb.com.localhost;
+  server_name  ~^(web\.keyman|keymanweb).com.localhost;
```
Comment (mcdurdin) [inline resources/nginx.conf:34]:
```suggestion
  server_name  ~^(web\.keyman|keymanweb)\.com\.localhost;
```

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**